### PR TITLE
Precache bundles

### PR DESCRIPTION
**Pull request by rogerfranklin2**
No tests around the precache as I'm not sure how we can mock out the config to set the `preCacheBundle`

### CODE REVIEW

File: script/preCache.js [hunk 1]
```diff
@@ -9,17 +9,21 @@ module.exports = breadboard({
     'package.json': require('../package.json')
   },
   entry: ({
+    '/config/index': getConfig,
     '/lib/bundler/index': bundle,
     '/lib/getComponentInfo': getComponentInfo
   }) => {
+    const config = getConfig();
     const components = getComponentInfo().map(componentInfo => componentInfo.name );
+    const preCacheComponentBundles = config.components && config.components.preCacheBundles ? config.components.preCacheBundles : [];
```
Comment (chrisface) [inline script/preCache.js]: Not sure what this line is trying to do. What's the `&&` for?

Comment (rogerfranklin2) [inline script/preCache.js]: The`&&` line is to check that `config.components` is there before we call `config.components.preCacheBundles`. This should be fine actually in real life just the tests were breaking if we didnt set config.components

Comment (rogerfranklin2) [inline script/preCache.js]: This line reads the config for the precache bundles which come from the toga.json from components

Comment (chrisface) [inline script/preCache.js]: That's completely clear now that I read it again. I'm not sure what I was thinking yesterday.

**Comment by chrisface** [comment]
👍

**Pull request merged**